README.adoc

```diff
@@ -1,7 +1,6 @@
 = CloudBees action: Store an object on Amazon S3
 
-Use this action to upload an object file to Amazon Simple Storage Service (Amazon S3).
-
```

Comment:
@Dawn-Davenport the removal of newline here has some unintended impact on the markdown.